### PR TITLE
Clean and update project

### PR DESCRIPTION
**Pull request by FrancoisDevemy5**
Apply fix from XCode warnings.
Update pods.
Delete unused SandboxUITests.
Fix formatting
Cleanup `return` and `self`

À relire commit par commit ou masquer les espaces blancs

### CODE REVIEW

File: IdentitySdkCore.podspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |spec|
   spec.homepage              = "https://github.com/ReachFive/identity-ios-sdk"
   spec.license               = { :type => "MIT", :file => "LICENSE" }
   spec.author                = "ReachFive"
-  spec.authors               = { "egor" => "egor@reach5.co", "guillaume" => "guillaume@reach5.co", "roxane" => "roxane@reach5.co", "Pierre" => "pierre.bar@reach5.co", "gbe" => "guillaume.bersac@reach5.co", "Matthieu" => "matthieu@reach5.co" }
+  spec.authors               = { "egor" => "egor@reach5.co", "guillaume" => "guillaume@reach5.co", "roxane" => "roxane@reach5.co", "Pierre" => "pierre.bar@reach5.co", "gbe" => "guillaume.bersac@reach5.co", "Matthieu" => "matthieu@reach5.co", "François" => "francois.devemy@reach5.co" }
```
Comment (axuan25) [inline IdentitySdkCore.podspec:13]: We should probably remove the other authors since they're effectively not reachable anymore.

**Review by axuan25 — APPROVED** [review]

File: IdentitySdkCore.podspec (hunk 1 above)
Comment (FrancoisDevemy5) [reply IdentitySdkCore.podspec:13]: ok

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift [hunk 2]
```diff
@@ -20,7 +20,7 @@ public class ReachFive: NSObject {
     internal var scope: [String] = []
     internal let storage: Storage
     let codeResponseType = "code"
-
+
```
Comment (thomasreach5) [inline IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift:23]: The formatter indents empty lines?

**Review by thomasreach5 — APPROVED** [review]

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift (hunk 2 above)
Comment (FrancoisDevemy5) [reply IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift:23]: Yes.
The style was very inconsistent, with a lot of code with these indented empty lines, and a lot of code without.
But it seems that indented empty lines was more frequent and older, so I decided to make it the default.
I could've gone the opposite way, but in the end I don't really care, just that the style is consistent.

**Pull request merged**